### ISSUE TITLE
After removing the last-opened flowgraph, a segfault occurs when starting up with `gnuradio-companion`

### ISSUE THREAD

**Issue by marcusmueller** [gnuradio/gnuradio#7589]
Extracted from #6957.

**Comment by marcusmueller** [comment]
@lukazlim said:

> 
> The `--gtk` version has no issues, it defaults to a new flowgraph which should be the intended behaviour.
> 
> ![image](https://github.com/gnuradio/gnuradio/assets/111743620/f62ddfff-9799-4f9f-a678-d0f6b3cdacb1)
> 
> Restoring/moving the flowgraph back to its original directory will stop those errors until the program is closed while on a new unsaved flowgraph.
> 
> Tested on Ubuntu 22.04.4 with Python 3.10.12

**Comment by haakov** [comment]
This one needs some investigation. It looks like the segfault is coming from GRC Gtk

**Comment by marcusmueller** [comment]
Correct, the Qt version doesn't print the `<<< Welcome to GNU Radio Companion` in the log level setting that @lukazlim seems to use. 

@lukazlim how are chances you would be able to reproduce the issue for us? I'd especially be interested in a *backtrace*, if that would be possible

In short: 

0. install `gdb` if you don't have that, 
1. run `gdb --args "$(head -n1 $(which gnuradio-companion) | sed 's;^[^/]*\(.*\)$;\1;')" "$(which gnuradio-companion)"` <br> (all this does is finding out which python interpreter *exactly* is in use to execute the `gnuradio-companion` python program, then invoking that with the path to the same program), then
2. on that "`(gdb)`" prompt :
    1. enable logging:  `set logging on`
    2. set where to log: `set logging file grc_issue_7589.txt`
    3. run GRC: `run`
    4. wait till segfault
    5. dump all threads' backtraces: `thread apply all backtrace`
    6. quit: `quit`
3. upload the `grc_issue_7589.txt` here
